cli/ssync/main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/rancher/sparse-tools/log"
```

Comment:
Should, but there are a lot of other places using it, such as sfold and all the ssync and sfold tests etc.